docs/approach.rst

```diff
@@ -191,22 +203,27 @@ TEDICA
 Next, ``tedana`` applies TE-dependent independent components analysis (ICA) in
 order to identify and remove TE-independent (i.e., non-BOLD noise) components.
 The dimensionally reduced optimally combined data are first subjected to ICA in
-order to fit a mixing matrix to the whitened data.
+order to fit a mixing matrix to the whitened data. This generates a number if 
+independent timeseries (saved as **meica_mix.1D**), as well as beta maps which show 
```

Comment:
Should we explain that the beta maps are derived from fitting a model to the classified components ?

Reply:
Ah, this is explained below ! Can we re-arrange ?